### PR TITLE
Sanitize leak

### PR DESCRIPTION
**Pull request by lionelgo**
Old work on leaks that seems it has not been merged

### CODE REVIEW

File: build/scripts/build_helper [hunk 1]
```diff
@@ -121,6 +121,7 @@ get_distribution_release() {
 check_supported_distribution() {
     local distribution=$(get_distribution_release)
     case "$distribution" in
+        "ubuntu20.04") return 0 ;;
```
Comment (rdefosse) [inline build/scripts/build_helper:124]: @lionelgo 

have you tried to build on a `Ubuntu20` server?

File: src/itti/itti.hpp [hunk 2]
```diff
@@ -240,7 +240,7 @@ class itti_mw {
    *  \param task_id task id of the task that requested the timer
    *  @returns -1 on failure, 0 otherwise
    **/
-  int timer_remove(timer_id_t& timer_id);
+  int timer_remove(const timer_id_t& timer_id);
```
Comment (rdefosse) [inline src/itti/itti.hpp:243]: @lionelgo 

what is the purpose of having a `const` on your timer_id? I am a bit rusty on my C/C++.

**Review by rdefosse — CHANGES_REQUESTED** [review]
Looks good.
I've asked a few questions for curiosity
just re-format the 5 files that were not formatted.

File: src/itti/itti.hpp (hunk 2 above)
Comment (lionelgo) [reply src/itti/itti.hpp:243]: Cv-qualified references are ill-formed except when the cv-qualifiers are introduced through the use of a typedef-name or decltype-specifier, in which case the cv-qualifiers are ignored.

Would have been a good idea to use const on a timer_id_t, not a good idea to use it on a timer_id_t&

**Review by rdefosse — APPROVED** [review]

**Pull request merged**